Aptoide/aptoide-client-v8
# [ASV-1720] Changed home bundles view holders to not use the onBind po…

**What does this PR do?**

This PR aims to change the way we pass viewholder positions to home bundles. Instead of using the position provided when onBind is called, we now call getAdapterPosition inside the viewHolder

**Database changed?**

  No

**Where should the reviewer start?**

- [ ] BundlesAdapter.java

**How should this be manually tested?**

The main implications for most of these positions regard analytics. Since these are currently impossible to check in charles, I used break points in every view holder onMoreClick method and checked if it matched. You can also add logs in the same place and check if the value is correct;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1720](https://aptoide.atlassian.net/browse/ASV-1720)

**Code Review Checklist**
